components/containers/cards/cardevent.stories.js

```diff
@@ -52,6 +52,6 @@ export const CardEvent = () => ({
 })
 
 export default {
-  title: 'Elements',
+  title: 'Containers',
```

Comment:
If we follow Atomic Design, shouldn't it be `Patterns`?

Reply:
sure. I like the term container here but yeah i should update it to be consistent

Reply:
`Patterns/Containers`?

Reply:
Stay consistent and let say patterns